x-pack/plugins/cases/common/utils/validators.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const isEmptyString = (value: string) => value.trim() === '';
```

Comment:
How do you feel about switching the name to `isOnlySpaces` or something like that, when I see `isEmptyString` my initial thought is that it's only checking for the string being empty, not that it's doing a `trim`.